roles/nova-data/tasks/libvirt.yml

```diff
@@ -56,29 +58,124 @@
   notify: restart nova services
   when: openstack_install_method == 'source'
 
+- name: make pki paths
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: kvm
+  with_items:
+    - /etc/pki/CA
+    - /etc/pki/libvirt/private
+
+- name: install CA file
+  template:
+    src: etc/pki/CA/cacert.pem.j2
```

Comment:
Shoot, I missed this in earlier review. We don't use the `.j2` convention for template names. We just name them the same extension as the target file.